creativecloud/blocks/sidenav/sidenav.js

```diff
@@ -4,31 +4,42 @@ import '../../deps/lit-all.min.js';
 import '../../deps/merch-spectrum.min.js';
 import '../../deps/merch-sidenav.js';
 
-const CATEGORY_TYPE = 'Categories';
-const TYPE_TYPE = 'Types';
-const getValueFromLabel = (content) => content
-  .trim()
-  .toLowerCase()
-  .replace(/( and )/g, ' ')
-  .replace(/-/g, '')
-  .replace(/\s+/g, '-');
+const CATEGORY_TYPE_PREFIX = 'categories/';
+const TYPE_TYPE_PREFIX = 'types/';
 
-const getCategories = (arrayCategories) => {
-  const tag = createTag('sp-sidenav', { variant: 'multilevel', manageTabIndex: true });
+const getValue = (id) => (id?.substring(id.lastIndexOf('/') + 1) || id);
```

Comment:
`id?` implies that `id` can be `null` while `id.lastIndexOf('/')` does not
if it can, then the latter should also contain `Elvis` operator

also I'm unsure if `getValue` is a good name for this function
it actually returns leaf item in the hierarchy